### ISSUE TITLE
Video viewer should render captions

### ISSUE THREAD

**Issue by tpendragon** [pulibrary/figgy#6222]
We're adding captions to the HLS manifest now, but Universal Viewer doesn't render them.

We need to identify a new viewer and get it loading for Video.

Options:

Clover: https://samvera-labs.github.io/clover-iiif/
Ramp: https://ramp.avalonmediasystem.org/

We're gonna do Clover.

**Comment by tpendragon** [comment]
I couldn't get ramp to render, something about it wanting a `process` to do the include. Sent a message to samvera slack, maybe they'll get back.

For Clover, https://github.com/pulibrary/figgy/compare/main...try-clover is a quick and dirty proof of concept. The sizing in the iFrame is a bit odd, but it does render and the captions work great.

We're going to need something in the GraphQL to tell us to use the video viewer. Creating a new ticket.

**Comment by tpendragon** [comment]
Blocked by #6233
